app/mentions/lib/MentionsParser.js

```diff
@@ -82,7 +82,7 @@ export class MentionsParser {
 			return this.userTemplate({
 				prefix,
 				className,
-				mention,
+				mention: mentionObj?.type === 'team' ? mentionObj?.roomId : mention,
```

Comment:
Sometimes when mentioning a team, mentionObj returns undefined. This is due to the `mentions` property of the message no containing the property `name`. I tried to find where this property is set but was unsuccessful.

Maybe backend?

Reply:
There's something related to the backend, the team is aware of that and they will fix it ASAP